src/server/routes/callback.js

```diff
@@ -396,7 +396,7 @@ export default async function callback(req, res) {
       ...cookies.sessionToken.options,
     })
 
-    await events.signIn({ user, account })
+    await events.signIn({ user, account, profile: null })
```

Comment:
Should be `userObjectReturnedFromAuthorizeHandler` (really sorry for the variable name... 😅) and so please update the documentation and TS types to reflect this. 😊

UPDATE:
Hmm. Revisited the code and actually that variable is used for `user` as well. Should we have the same value twice, or remove profile and mention in the docs that they have user? What does make the most sense in your opinion?

Reply:
My mental model associates `profile` with "the thing the auth platform returns", so IMO nulling it out and pointing them to `user` makes more sense.

I'm also _relatively new here_ and _don't use the credentials flow at all_ so I have no strong opinions here

Reply:
Understandable! In that case, I think I would just drop `profile: null` and say in the docs that `user` is what they are looking for if it's a `credentials` provider. If for some reason we add `profile` later on, we could do it in a non-breaking manner then.

Reply:
Just updated @balazsorban44 , ty for the feedback!